src/fpsgame/server.cpp

```diff
@@ -2601,12 +2605,12 @@ namespace server
         return ci && ci->connected;
     }
 
-    clientinfo *findauth(uint id)
-    {
-        loopv(clients) if(clients[i]->authreq == id) return clients[i];
-        return NULL;
-    }
-
+    // remod: "overriden" in authservers.cpp
```

Comment:
Commenting out all these auth functions isn't strictly necessary, but felt right to make sure they aren't used anymore. let me know if you think otherwise and I'll leave them untouched.

Reply:
Comment unused functions is not neccesary, i prefer keep vanilla code unmodified as much as possible. Sorry for delay, last week i was on holidays

Reply:
Okay: aa44c397c7aadca0f5dabade407610640277f9ff